src/lib/gundb/UserStorageClass.js

```diff
@@ -2076,7 +2074,7 @@ export class UserStorage {
 
   //eslint-disable-next-line
   async _extractAvatar(type, withdrawStatus, profileToShow, address) {
-    const favicon = `${process.env.PUBLIC_URL}/favicon-96x96.png`
+    const favicon = `${env.PUBLIC_URL}/favicon-96x96.png`
```

Comment:
@MathiasMoralesDixtra I don't think this is gonna work in mobile, let's apply this same fix in react-native and make sure it works before we push this to master

Reply:
Fixed